examples/nodejs/aws/eventbridge/deploy-and-run-cli.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+set -e
+set -x
+
+# Check if .env file exists
+if [ ! -f .env ]; then
+    echo ".env file not found. See README.md for more information: https://github.com/momentohq/client-sdk-javascript/tree/main/examples/nodejs/aws/eventbridge/webapp/README.md"
+    exit 1
+fi
+
+# Load environment variables from .env.development file
+set -a
+source .env
+set +a
+
+# Check if environment variables are loaded
+if [ -z "$MOMENTO_API_KEY" ] || [ -z "$AWS_ACCESS_KEY_ID" ] || [ -z "$AWS_SECRET_ACCESS_KEY" ] || [ -z "$AWS_REGION" ]; then
+    echo "Error: Environment variables not loaded from .env file."
```

Comment:
these error messages do not tell the user which env vars are missing or how to fix it.